package.json

```diff
@@ -17,7 +17,7 @@
     "test:int": "jest ./tests/integration",
     "test:esm": "jest ./tests/esm",
     "test:u": "yarn test:jest -u && yarn test:badge",
-    "test:types": "tsc --noEmit",
+    "test:types": "tsc",
```

Comment:
`noEmit` moved to tsconfig.
it's needed to fix TS error on using `allowImportingTsExtensions` option without it

Reply:
reverted